common/json.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('node:fs');
```

Comment:
And the same here!

Reply:
More shared code. Read: CLI and stage 4. Write: CLI and stage 2.